### DISCUSSION TITLE
📝 [FR / Bug ?] When using `biome migrate prettier --write`, add the Prettier default of removing JSON trailing commas

### DISCUSSION THREAD

**Discussion by MarArMar** [biomejs/biome#3439] [Ideas]
CF https://github.com/biomejs/biome/issues/926

I just migrated from prettier and did not have the behavior and was surprised

Config is :
```
  "json": {
    "parser": {
      "allowTrailingCommas": true,
    },
    "formatter": {
      "trailingCommas": "none"
    }
  }
```

### Playground link

not needed

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
What's the reasoning for adding that?

**Reply by MarArMar** [reply]
Isn't it the default of prettier ?

**Reply by ematipico** [reply]
I don't know! Hence my question :)

**Reply by MarArMar** [reply]
From what I teste : it is actually the stock VSCode JSON formatter extension "vscode.json-language-features" that was removing the trailing commas

Basically I used it before, it removed the commas, now I installed Biome, set in my settings 

```
    "[jsonc]": {
      "editor.defaultFormatter": "biomejs.biome"
    },
    "[json]": {
      "editor.defaultFormatter": "biomejs.biome"
    },
```

and it stopped removing it and by default it was also not working because of an extra comma

Maybe add then a documentation to migrate from VSCode JSON Formatter that I believe many people also use by default ?

**Reply by MarArMar** [reply]
Anyway because we have this conversation it will be available for people on Google with SEO so that is good anyway

**Reply by ematipico** [reply]
I believe we already document this part very well. Is there something missing from this section? https://biomejs.dev/internals/language-support/#jsonc-support

**Reply by MarArMar** [reply]
Ah ok but I believe it was not well referenced on google because I must have types "JSON" not JSONC and I also searched for JSON in biome and did not find I think

I ended up in a Github issue asking people
